dart/examples/io_example/main.dart

```diff
@@ -25,10 +25,22 @@ Future<void> main(List<String> rawArgs) async {
   // Sends a full Sentry event payload to show the different parts of the UI.
   final sentryId = await Sentry.captureEvent(event);
```

Comment:
can we add the missing methods here before capturing events?

eg
```
Sentry.addBreadcrumb(...)
Sentry.configureScope(scope -> {
  scope.user = User(...)
  ...
})
```

Reply:
so all the following events will have an user, a breadcrumb, an extra, a tag, etc...

Reply:
✅